.github/workflows/CODEOWNERS

```
@@ -0,0 +1,2 @@
+* oleksandr-pavlyk
+* ndgrigorian
```

```suggestion
*       @oleksandr-pavlyk @ndgrigorian
```
According to the [Github documentation on codw owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) I think it should look something like this, currently the file says both are invalid